components/color-picker/components/ColorPresets.tsx

```diff
@@ -44,7 +46,13 @@ const ColorPresets: FC<ColorPresetsProps> = ({ prefixCls, presets, value: color,
   const colorPresetsPrefixCls = `${prefixCls}-presets`;
 
   const activeKeys = useMemo<string[]>(
```

Comment:
<img width="855" alt="image" src="https://github.com/ant-design/ant-design/assets/32004925/844b477d-2a60-4552-be64-ab5215504408">

```suggestion
  const activeKeys = useMemo(
```

Reply:
还是不加防御了吧，这里就交给开发者自己双非 (!!) 处理